### DISCUSSION TITLE
Is it any possible integrate some functions like htmx?

### DISCUSSION THREAD

**Discussion by athxx** [alpinejs/alpine#3789] [3. Feature ideas]
alpine.js is the greatest js that I've used, and recently I try to develop some function that I have to write js to control request, and I found htmx. I think alpine.js and htmx is the best match, if both framework can integrate will be awesome. Is it any possible integrate some functions like htmx to reduce write lots of js request?

**Comment by ekwoka** [comment]
there is [alpine-ajax](https://github.com/imacrayon/alpine-ajax) that can help with quite a lot of this.
